Tests/SonarScanner.Shim.Tests/PropertiesFileGeneratorTests_MultiTarget.cs

```diff
@@ -0,0 +1,364 @@
+            var result = generator.GenerateFile();
+
+            // Assert
+            AssertExpectedProjectCount(1, result);
+
+            // One valid project info file -> file created
+            AssertPropertiesFilesCreated(result, logger);
+            var propertiesFileContent = File.ReadAllText(result.FullPropertiesFilePath);
+            AssertFileIsReferenced(files[0], propertiesFileContent);
+            AssertFileIsReferenced(files[1], propertiesFileContent);
+        }
+
+        [TestMethod]
+        public void Different_Files_Per_Target_The_Ignored_Compilations_Are_Not_Included()
+        {
+            var guid = Guid.NewGuid();
+
+            /*  Solution1/
+                    .sonarqube/
+                        conf/
+                        out/
+                            Project1_Debug/
+                                ProjectInfo.xml
+                                FilesToAnalyze.txt (file1.cs)
+                            Project1_Release/ // ignored, file2.cs should not be included in the properties file
+                                ProjectInfo.xml
+                                FilesToAnalyze.txt (file2.cs)
+                    Project1/
+                        file1.cs // included in Debug
+                        file2.cs // included in Release
+            */
+
+            // Arrange
+            var solutionDir = TestUtils.CreateTestSpecificFolder(TestContext, "Solution1");
+
+            List<string> files;
+            var projectRoot = CreateProject(solutionDir, out files);
+
+            var outDir = Path.Combine(solutionDir, ".sonarqube", "out");
+            Directory.CreateDirectory(outDir);
+
+            // Create two out folders for each configuration, the included files are different
+            CreateProjectInfoAndFilesToAnalyze(guid, "Debug", files.Take(1), projectRoot, outDir);
+            CreateProjectInfoAndFilesToAnalyze(guid, "Release", files.Skip(1), projectRoot, outDir, ignored: true);
+
+            var logger = new TestLogger();
+            var config = new AnalysisConfig()
+            {
+                SonarProjectKey = guid.ToString(),
+                SonarProjectName = "project 1",
+                SonarOutputDir = outDir,
+                SonarProjectVersion = "1.0",
+            };
+
+            var generator = new PropertiesFileGenerator(config, logger);
+
+            // Act
+            var result = generator.GenerateFile();
+
+            // Assert
+            AssertExpectedProjectCount(1, result);
+
+            // One valid project info file -> file created
+            AssertPropertiesFilesCreated(result, logger);
+            var propertiesFileContent = File.ReadAllText(result.FullPropertiesFilePath);
+            AssertFileIsReferenced(files[0], propertiesFileContent);
+            AssertFileIsReferenced(files[1], propertiesFileContent, 0);
+        }
+
+        [TestMethod]
+        public void All_Targets_Ignored_Compilations_Are_Not_Included()
```

Comment:
Could you add some test with a .net standard structure (e.g. bin/debug/net46)?

Reply:
I reverted the change of the Disambiguator and it is using the timestamp again. The output from all configurations is flat sub-directories in the out folder